packages/react-component-library/src/components/RangeSliderE/ThresholdTrackE.tsx

```diff
@@ -80,7 +80,7 @@ export const ThresholdTrackE: React.FC<ThresholdTrackEProps> = ({
       <ChunkE
         $thresholdColor={RANGE_SLIDER_TRACK_ABOVE_THRESHOLDS}
         $left={thresholds[thresholds.length - 1]}
-        $width={target.percent - thresholds[thresholds.length - 1]}
+        $width={Math.max(0, target.percent - thresholds[thresholds.length - 1])}
```

Comment:
Fixes issue where a single threshold would remian visible depsite not being within range:

https://user-images.githubusercontent.com/66470099/144870716-90cf4491-aedd-4321-b6e7-3fdd740d159c.gif